okhttp/src/commonTest/kotlin/okhttp3/MediaTypeTest.kt

```diff
@@ -1,6 +1,5 @@
 /*
- * Copyright (C) 2013 Square, Inc.
- * Copyright (C) 2011 The Guava Authors
+ * Copyright (C) 2022 Square, Inc.
```

Comment:
Is this copyright still warranted? cc @swankjesse

Reply:
yes!